node/clients/externalPrice.ts

```diff
@@ -16,12 +16,16 @@ export default class ExternalPrice
   }
 
   public async getPrice(item: InputItem): Promise<number | undefined> {
-    const result = await this.http.get(item.skuId, {
-      metric: 'get-price',
-    })
+    try {
+      return await this.http.get(item.skuId, {
+        metric: 'get-price',
+      })
+    } catch (e) {
+      if (e.response.status === 404) {
+        return undefined
```

Comment:
quick question, on json serialization a undefined value will omit the entry or it will create a key and a null value? What is the expected behavior?

Reply:
I didn't change the implementation, but it is checked if the price is undefined to throw a NotFoundException https://github.com/vtex/unilever-external-prices-node/blob/main/node/handlers/fetchPrice.ts#L36.

Reply:
okay, so in this case null or undefined will have the same behavior at this point.